assets/javascripts/comments.js

```diff
@@ -100,6 +100,12 @@ function updateComment(form) {
         textElement.val(text);
         markdownElement.html(markdown);
         showCommentEditor(form);
+        var responseJSON = xhr.responseJSON;
+        var flashTarget = $('#flash-messages-finished-jobs');
+        if (responseJSON && responseJSON.error) {
+          addFlash('danger', '<strong>Unable to add comment' + responseJSON.error + '</strong>', flashTarget);
```

Comment:
```suggestion
          addFlash('danger', '<strong>Unable to add comment' + responseJSON.error + '</strong>', $('#flash-messages-finished-jobs'));
```
Instead of the extra variable. Additionally, you can use `const`/`let` in new code.